lib/improver/tests/uv_index/test_uv_index.py

```diff
@@ -91,9 +91,11 @@ def test_scale_factor(self):
 
     def test_metadata(self):
         """ Tests that the uv index output has the correct metadata (no units,
-        and name = uv index)."""
+        and name = ultraviolet index)."""
         result = calculate_uv_index(self.cube_uv_up, self.cube_uv_down)
         self.assertEqual(str(result.standard_name), 'ultraviolet_index')
+        self.assertEqual(str(result.var_name), 'None')
+        self.assertEqual(str(result.long_name), 'None')
```

Comment:
Do these have to be made into strings, of can you just ```self.assertEqual(result.var_name, None)```?

Reply:
Or use ```self.assertIsNone```

Reply:
I went for self.assertIsNone, and that should all work now.